### DISCUSSION TITLE
v2/copy-code/zh/config

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#368] [Announcements]
# v2/copy-code/zh/config

selector 类型: string | string[]; 默认值: '.theme-default-content div[class*="language-"] pre'; 代码块选择器 duration 类型: number; 默认值: 2000; 提示消息显示时间，设置为 0 会禁用提示。 showInMobile 类型: boolean;...

https://vuepress-theme-hope.github.io/v2/copy-code/zh/config.html

**Comment by HF10** [comment]
白色背景的时候字体是黑色的，但是代码块的颜色是黑色的

**Reply by Mister-Hope** [reply]
Needs further eplaination and maybe screenshots.

**Comment by qian357891** [comment]
The same problem，The style of copied prompt text will change with the dark/bright mode：
dark mode is works：
![image-20221119232337359](https://qiankun825.oss-cn-hangzhou.aliyuncs.com/img/image-20221119232337359.png)

light mode：
![image-20221119232733683](https://qiankun825.oss-cn-hangzhou.aliyuncs.com/img/image-20221119232733683.png)
